cas/utils.go

```diff
@@ -72,3 +77,25 @@ func decompress(rs io.Reader, typ aci.FileType) (io.Reader, error) {
 	}
 	return dr, nil
 }
+
+// ExtractImageManifest returns, given the blockStore key, the
+// schema.ImageManifest of an ACI
+func ExtractImageManifest(key string, ds *Store) (*schema.ImageManifest, error) {
+	rs, err := ds.ReadStream(key)
+	defer rs.Close()
+	if err != nil {
+		return nil, fmt.Errorf("error extracting ImageManifest: %v", err)
+	}
+
+	imb, err := ptar.ExtractFileFromTar(tar.NewReader(rs), "manifest")
```

Comment:
Can we use aci.ManifestFromImage here:
 https://github.com/appc/spec/blob/master/aci/file.go#L112